### ISSUE TITLE
Arm64 makefile bug

### ISSUE THREAD

**Issue by CodeOriginal** [kirb/theos#1]
When compiling on iPhone 5s, iOS 8.1.2, using iOS 8.1 sdk, with makefile:

```
export ARCHS = arm64
export TARGET = iphone:clang:8.1:8.1

include theos/makefiles/common.mk

TWEAK_NAME = PowerInject
PowerInject_FILES = Tweak.xm


include $(THEOS_MAKE_PATH)/tweak.mk

internal-stage::
    #PreferenceLoader plist
    $(ECHO_NOTHING)if [ -f Preferences.plist ]; then mkdir -p $(THEOS_STAGING_DIR)/Library/PreferenceLoader/Preferences/PowerInject; cp Preferences.plist $(THEOS_STAGING_DIR)/Library/PreferenceLoader/Preferences/PowerInject/; fi$(ECHO_END)

after-install::
        install.exec "killall -9 SpringBoard"
```

I get the error:

```
iphone: root# make messages=YES
    make: xcode-select: Command not found
    make: xcode-select: Command not found
```

And upon changing 'iphone' to 'iPhone' I get: 

```
iphone: #root make messages=YES
    theos/makefiles/common.mk:117: *** the "iPhone" target is not supported on the "macosx" platform. STOP. 
```

My guess is that it is an issue somewhere between where common.mk gets the platform with 'uname -s', 'uname -p', and 'uname -o'; which return 'Darwin', 'arm64', and 'Darwin' respectively when run from the command line. 

Right now, CoolStar's fork seems to be the only one working with device.

**Comment by CodeOriginal** [comment]
Fixed in commit [5958cdf](https://github.com/kirb/theos/commit/5958cdf8e579c3a8c9a8d7fb7c1d8f0139a8defc)

**Comment by kirb** [comment]
(Didn't see you filed an issue – thanks!)

**Comment by CodeOriginal** [comment]
Not entirely fixed, to fix it on my device, I created a symlink 'Darwin-arm64.mk' in 'theos/makefiles/platform' to 'Darwin-arm.mk'.

However, I believe it is still broken, because now I get the error: "theos/makefiles/targets/Darwin/iphone.mk:23: **\* first argument to 'word' function must be greater than 0.  Stop." Even though I have the sdk in the correct folder and am using the correct TARGET in my makefile. 

The fact that it is still acting as though my iPhone is on the macosx platform is still concerning. (In the above error, the error is in the "Darwin/iPhone.mk" instead of the "Darwin-arm" or "Darwin-arm64" version of "iphone.mk")

**Comment by CodeOriginal** [comment]
From irc:

22:57 <~CodeOriginal> kirb: the commit on github fixed the platform includes, but not the target includes, the line with the issue is common.mk:119, which should be below the code on common.mk:122 and the same movement should be done for the evals below them.

22:58 <~CodeOriginal> Also, your fork needs the symlink in makefiles/platform for Darwin-arm64

**Comment by kirb** [comment]
Arg, thanks. Fixed (I hope) in 5958cdf8e579...be0388e25bbd.

**Comment by CodeOriginal** [comment]
Works, thanks!
